plugin/core/transports.py

```diff
@@ -281,7 +281,7 @@ def start_subprocess() -> subprocess.Popen:
         config.name, process, sock, reader, writer, stderr, json_rpc_processor, callback_object)  # type: ignore
 
 
-_subprocesses = weakref.WeakSet()  # type: weakref.WeakSet[subprocess.Popen]
+_subprocesses: 'weakref.WeakSet[subprocess.Popen]' = weakref.WeakSet()
```

Comment:
note to myself to check why it was necessary to use `'weakref.WeakSet[subprocess.Popen]'` instead of `weakref.WeakSet[subprocess.Popen]`

Reply:
`WeakSet` and `WeakValueDictionary` are not generic types like classes from typing like for example `List`, so you can't use the notation of having another type inside square brackets on them.

It would work with `from __future__ import annotations`, because then type hints are automatically turned into strings.

Reply:
Thanks!